flink-table/flink-table-planner-blink/src/test/resources/org/apache/flink/table/planner/plan/stream/sql/MiniBatchIntervalInferTest.xml

```diff
@@ -222,7 +222,7 @@ LogicalProject(a=[$0], b=[$1])
     <Resource name="planAfter">
       <![CDATA[
 Calc(select=[a, b])
-+- Join(joinType=[InnerJoin], where=[=(a0, a1)], select=[a, b, a0, b0, a1], leftInputSpec=[HasUniqueKey], rightInputSpec=[HasUniqueKey])
++- Join(joinType=[InnerJoin], where=[=(a0, a1)], select=[a, b, a0, b0, a1], leftInputSpec=[JoinKeyContainsUniqueKey], rightInputSpec=[HasUniqueKey])
```

Comment:
`JoinKeyContainsUniqueKey` is correct, because `a` is the group by key of left stream and also the join field. And `a` is an int type, `cast(a as bigint)` doesn't lose information so that it keeps the unique key information of left stream.